demo-v1/test/spec/components/screens/DialingScreen-test.tsx

```diff
@@ -0,0 +1,57 @@
+import { fireEvent, screen } from "@testing-library/react";
+import DialingScreen from "../../../../src/components/screens/DialingScreen";
+import { ScreenNames } from "../../../../src/types/ScreenTypes";
+import { renderWithContext } from "../../../render";
+
+const noop = (..._args: any[]) => {};
+
+const cti = {
+  callAnswered: noop,
+};
+
+const props = {
```

Comment:
Seeing these repeated in all screen tests, what do you think about creating a `getScreenProps` test util?

Reply:
Since `props.cti` is different for each screen test and it's nice to have the default props in the same place as the tests, I am not sure if the we need to eliminate duplication here